eventmesh-protocol-plugin/eventmesh-protocol-http/src/main/java/org/apache/eventmesh/protocol/http/HttpProtocolAdaptor.java

```diff
@@ -37,6 +37,7 @@
 import java.util.Map;
 import java.util.Objects;
 import java.util.Set;
+import java.util.Collections;
```

Comment:
This import should be placed here:
![image](https://user-images.githubusercontent.com/91315508/236716064-6566c68c-f187-4f93-ac83-06f393743d5b.png)